### PR TITLE
Define processing model

### PR DESCRIPTION
**Pull request by marcoscaceres**
***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/payment-method-basic-card/notes_to_normative.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/payment-method-basic-card/2e4156f...1f6b2d3.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -96,6 +94,27 @@
         "<code><dfn>basic-card</dfn></code>".
       </p>
     </section>
+    <section>
```
Comment (marcoscaceres) [inline index.html]: IGNORE THIS SECTION FOR NOW.

**Comment by marcoscaceres** [comment]
Ok, this is ready for review.

**Comment by marcoscaceres** [comment]
20+ tests incoming.

File: index.html [hunk 2]
```diff
@@ -114,31 +146,65 @@
...
-        matches one of the types.
+    </section>
+    <section data-link-for="BasicCardRequest">
+      <h2>
+        Interfacing with a payment handler
+      </h2>
+      <p>
+        The <dfn>steps to constrain a payment handler</dfn> with
+        <var>data</var> <a>BasicCardRequest</a> are given by the following
+        algorithm. If the end-user inputs or selects a <a>card</a> that meets
```
Comment (sideshowbarker) [inline index.html]: algorithm. If the end-user inputs or selects a <a>card</a> that meets

“end-user” doesn’t need a hyphen; should just be “end user”

File: index.html [hunk 3]
```diff
@@ -114,31 +146,65 @@
...
+        <li>
+          <p>
+            Let <var>card</var> be the result of the end user selecting or
+            inputing a <a>card</a> of <a>type</a>
+            <var>data</var>.<a>supportedTypes</a>, or any type if
+            <var>data</var>.<a>supportedTypes</a> is empty, and from one of the
+            <a>networks</a> in <var>cardNetworks</var>, or from any network if
+            <var>cardNetworks</var> is empty. User agents SHOULD validate card
+            details to make sure they adhere to the <a>type</a> and
+            <a>network</a> requirements (e.g., checking that "visa" card starts
```
Comment (sideshowbarker) [inline index.html]: <a>network</a> requirements (e.g., checking that "visa" card starts

checking that **a** "visa" card starts

File: index.html [hunk 4]
```diff
@@ -114,31 +146,65 @@
...
+            with the number "4", etc.).
+          </p>
+          <div class="note" title="Validation of inputs">
+            <p>
+              The validation a user agent performs on the card inputs is a
+              "quality of implementation" detail and outside the scope of this
+              specification. There is nevertheless an expectation that user
+              agents will make a best effort to validate that card number via
```
Comment (sideshowbarker) [inline index.html]: agents will make a best effort to validate that card number via

“validate <s>that</s> **each** card number”

File: index.html [hunk 5]
```diff
@@ -241,15 +299,30 @@
         Security and Privacy Considerations
       </h2>
       <p>
+        Due to differences in "quality of implementation", and the end-user's
```
Comment (sideshowbarker) [inline index.html]: Due to differences in "quality of implementation", and the end-user's

I think it’s not necessary to put _quality of implementation_ in quotes here

File: index.html [hunk 6]
```diff
@@ -173,19 +177,20 @@
...
-            <a>networks</a> in <var>cardNetworks</var>, or from any network if
-            <var>cardNetworks</var> is empty. User agents SHOULD validate card
-            details to make sure they adhere to the <a>type</a> and
-            <a>network</a> requirements (e.g., checking that "visa" card starts
-            with the number "4", etc.).
+            Let <var>card</var> be the result of the end user inputting a
+            <a>card</a> of <a>type</a> <var>data</var>.<a>supportedTypes</a>,
+            or any type if <var>data</var>.<a>supportedTypes</a> is empty.
+            Check that <var>card</var> is from one of the <a>networks</a> in
+            <var>cardNetworks</var>. Validate <var>card</var>'s <a>details</a>
```
Comment (sideshowbarker) [inline index.html]: <var>cardNetworks</var>. Validate <var>card</var>'s <a>details</a>

Unless I’m misunderstanding the `Validate … to make sure they adhere to any <a>type</a> and/or <a>network</a> requirements` isn’t not clearly normatively defined and if so is not implementable and should be dropped. Or else this should be replaced with language that is implementable (e.g., using a reference to a normative definition of the *validate* intended here).

**Review by sideshowbarker — COMMENTED** [review]
This change LGTM

File: index.html [hunk 7]
```diff
@@ -103,6 +103,10 @@
         Model
       </h2>
       <p>
+        This section concepts used in this specification, and how those
```
Comment (sideshowbarker) [inline index.html]: This section concepts used in this specification, and how those

Missing a word there. “This section **defines** concepts used in this specification” I guess

**Review by sideshowbarker — COMMENTED** [review]
This change LGTM

**Pull request merged**